cmd/serverless/main.go

```diff
@@ -252,9 +252,8 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 
 	logChannel := make(chan *logConfig.ChannelMessage)
 
-	forwarderTimeout := config.Datadog.GetDuration("forwarder_timeout") * time.Second
 	metricAgent := &metrics.ServerlessMetricAgent{}
-	metricAgent.Start(forwarderTimeout, &metrics.MetricConfig{}, &metrics.MetricDogStatsD{})
```

Comment:
This refers to the timeout on the metrics agent forwarder (not to be confused with the Datadog Logs Forwarder). Previously, it was set using a config value (not specific to the Serverless agent). With this PR, it will be hardcoded to be the same as the overall flush timeout.